src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,341 @@
+class Guides::Frontend::Internationalization < GuideAction
+  guide_route "/frontend/internationalization"
+
+  def self.title
+    "Internationalization - Handle Multiple Languages"
```

Comment:
This wraps oddly in the sidebar so I think just "Internationalization" is ok

Reply:
yup fixed